### PR TITLE
Minor improvements to @frontity/google-ad-manager page

### PR DESCRIPTION
**Pull request by rmartinezduque**

### CODE REVIEW

File: docs-api/frontity-packages/features-packages/google-ad-manager.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ This package enables Frontity to integrate with Google Ad Manager. It allows you
     - [The `props` property](google-ad-manager.md#the-props-property)
 - [Examples](google-ad-manager.md#examples)
 - [Usage](google-ad-manager.md#usage)
-  - [Using the `Slot & Fill` pattern](google-ad-manager.md#using-the-slot-fill-pattern)
+  - [Using the `Slot & Fill` pattern](google-ad-manager.md#using-the-slot--fill-pattern)
```
Comment (mburridge) [inline docs-api/frontity-packages/features-packages/google-ad-manager.md:17]: Should this link be: google-ad-manager.md#using-the-slot-and-fill-pattern ?

**Review by mburridge — CHANGES_REQUESTED** [review]
I think the link on line 17 should be: google-ad-manager.md#using-the-slot-and-fill-pattern

File: docs-api/frontity-packages/features-packages/google-ad-manager.md (hunk 1 above)
Comment (rmartinezduque) [reply docs-api/frontity-packages/features-packages/google-ad-manager.md:17]: Yes, sorry Michael. It should be `google-ad-manager.md#using-the-slot-and-fill-pattern`.

**Review by mburridge — APPROVED** [review]

**Comment by mburridge** [comment]
Merged, thanks @rmartinezduque

**Pull request merged**